robot_A: 5
robot_B: 4
robot_C: 3
robot_D: 2
robot_E: 1